src/jit/codegencommon.cpp

```diff
@@ -10302,6 +10302,22 @@ void CodeGen::genCaptureFuncletPrologEpilogInfo()
 /*****************************************************************************
  *
  *  Generates code for an EH funclet prolog.
+ *
+ *
+ *  Funclets have the following incoming arguments:
+ *
+ *      catch/filter-handler: eax = the exception object that was caught (see GT_CATCH_ARG), ecx = FP
```

Comment:
Funclets on all other architectures get the argument via regular argument registers. Would it make more sense to pass the exception object in edx instead?